txt2tags.py

```diff
@@ -1898,9 +1901,12 @@ def Readfile(file_path):
 
 def Savefile(file_path, lines):
     try:
-        with open(file_path, "w") as f:
+        with io.open(file_path, "w", encoding="utf-8") as f:
             for line in lines:
-                f.write(line + "\n")
+                if is_python2:
+                    f.write((line + "\n").decode("utf-8"))
```

Comment:
I'd add the newline (u"\n") after the conversion.

Reply:
I'm surprised we need to decode here. Shouldn't the text already be unicode?

Reply:
Since txt2tags in Python 2 never handled Unicode text explicitly, I gues it was converted back to `str()` during some  step in the processing.

```console
$ python2 ../txt2tags.py bugs/table-empty.t2t 
Sorry! Txt2tags aborted by an unknown error.
Please send the following Error Traceback to the author (jendrikseipp@gmail.com):

Traceback (most recent call last):
  File "../txt2tags.py", line 5080, in <module>
    exec_command_line()
  File "../txt2tags.py", line 5073, in exec_command_line
    convert_this_files(infiles_config)
  File "../txt2tags.py", line 4419, in convert_this_files
    finish_him(outlist, myconf)
  File "../txt2tags.py", line 3907, in finish_him
    Savefile(outfile, outlist)
  File "../txt2tags.py", line 1906, in Savefile
    f.write(line + "\n")
TypeError: write() argument 1 must be unicode, not str
```